graphprot/GraphGen.py

```diff
@@ -52,15 +52,15 @@ def __init__(self, pdb_path, ref_path, graph_type='residue', pssm_path=None,
     @staticmethod
     def _get_pssm(pssm_path, mol_name, base_name):
 
-        pssmA = os.path.join(pssm_path, mol_name+'.A.pdb.pssm')
-        pssmB = os.path.join(pssm_path, mol_name+'.B.pdb.pssm')
+        pssmA = os.path.join(pssm_path, mol_name+'.A.pssm')
+        pssmB = os.path.join(pssm_path, mol_name+'.B.pssm')
```

Comment:
we should rename the files we give in example then Or at least check that the example still work